addons/stock/models/res_company.py

```diff
@@ -22,6 +22,10 @@ def _default_confirmation_mail_template(self):
         domain="[('model', '=', 'stock.picking')]",
         default=_default_confirmation_mail_template,
         help="Email sent to the customer once the order is done.")
+    annual_inventory_date = fields.Date(
```

Comment:
I think we should add a cron to update it annualy

Reply:
hmm ok it's working without the cron but we should only set the month and day instead

Reply:
C'est chiant, but it's done. It now needs to check the logic for the `annual_inventory_day` for every location now... maybe there's a better way to do it, but I couldn't think of one due to the annoyance of tracking month/day combinations each time one of them changes.